packages/datatrak-web/src/views/AccountSettingsPage/ChangePasswordSection/ChangePasswordForm.tsx

```diff
@@ -54,11 +53,16 @@ const StyledFormInput = styled(FormInput)`
 
 const StyledButton = styled(Button)`
   ${({ theme }) => theme.breakpoints.up('sm')} {
-    // HACK: Make button height match adjacent FormInput
+    /* HACK: Align button with adjacent FormInput, even when FormInput in error state */
+    margin-block-start: calc(1.125rem + 3px);
+    //                                  ^~~ margin-bottom of .MuiFormLabel-root
+    //                       ^~~~~~~~ line-height of .MuiFormLabel-root
+
+    /* HACK: Make button height match adjacent FormInput */
+    padding: 1rem;
     .MuiButton-label {
       border: 1px solid transparent;
       height: 1.1876em;
```

Comment:
This is minor but probably worth using `rems` so we don't get that confusing thing where `em` relates to the parent's font size as opposed to the overall font size

Reply:
Would normally completely agree, but in this case my goal is just to exactly match the layout behaviour of the text field we’re trying to line up with. I’ve commented it so it’s more clearly intentional 🙏